Gruntfile.js

```diff
@@ -114,10 +114,6 @@ module.exports = function(grunt) {
                 tpl = Handlebars.compile(content),
                 tplVars = {
                   browserName: browserName,
-
-                  // To include accessibility check in the test, pass in -a11y option, i.e.
-                  // "grunt test-e2e --browserName chrome -a11y"
-                  CHECK_A11Y: !!grunt.option('a11y')
```

Comment:
This flag was never used - so we can delete it in favor of hooking into `testcafe`